db/migrate/20220830164121_create_join_table_recipes_foods.rb

```diff
@@ -0,0 +1,11 @@
+class CreateJoinTableRecipeFoods < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipe_foods do |t|
+      t.decimal :quantity
+      t.references :recipe, null: false, foreign_key: true
+      t.references :food, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Kindly fix the naming of this migration file. While trying to migrate the the tables, I got this error:
![Capture](https://user-images.githubusercontent.com/85095702/188265014-2129007e-e42a-4fae-8709-245b0df23f5c.PNG)